lib/features/modeling/cmd/UpdateModdlePropertiesHandler.js

```diff
@@ -22,21 +22,27 @@ import {
  * @implements {CommandHandler}
  *
  * @param {ElementRegistry} elementRegistry
+ * @param {Translate} translate
  */
-export default function UpdateModdlePropertiesHandler(elementRegistry) {
+export default function UpdateModdlePropertiesHandler(elementRegistry,translate) {
   this._elementRegistry = elementRegistry;
+  this._translate = translate;
 }
 
-UpdateModdlePropertiesHandler.$inject = [ 'elementRegistry' ];
+UpdateModdlePropertiesHandler.$inject = [
+  'elementRegistry',
+  'translate'
+];
 
 UpdateModdlePropertiesHandler.prototype.execute = function(context) {
 
   var element = context.element,
       moddleElement = context.moddleElement,
-      properties = context.properties;
+      properties = context.properties,
+      translate = this._translate;
 
   if (!moddleElement) {
-    throw new Error('<moddleElement> required');
+    throw new Error(translate('<moddleElement> required'));
```

Comment:
Are you sure you want to translate technical error messages? These are not meant to appear to the end user in any way, but log to the console.

Reply:
I see a lot of throw new Errors with translations, for example: [    throw new Error(translate('element required'));](https://github.com/bpmn-io/bpmn-js/blob/5d1c447e87e398327845dce14b3319187845c63b/lib/features/modeling/cmd/UpdatePropertiesHandler.js#L91)

Reply:
You're right. I'll get this to the team to discuss.

Reply:
Some work in progress: We consider removing translation of technical errors - https://github.com/bpmn-io/bpmn-js/pull/2145.

Reply:
Decided (cf. https://github.com/bpmn-io/bpmn-js/pull/2145) - We're removing translation of technical errors.